### ISSUE TITLE
Training DeepLab on new images

### ISSUE THREAD

**Issue by JustinLiang** [martinkersner/train-DeepLab#13]
I'm trying to train deeplabv2_resnet101 on new images with only 3 labels (background + 2 other classes) so I set `NUM_LABELS=3`. I noticed that during training, the loss would go down but then it converges quickly when the accuracy of each layer is still low. These images are all 1000x1000 in size. All I did was change the folder paths in the script. I also tried different batch sizes and learning rates and it seems like they all converge to the same loss value. Any idea what could be going on here?

**Comment by SJTUsuperxu** [comment]
hello @JustinLiang ,
    I am also training the DeepLab model on images with 2 classes but I cannot make it because it always throws an error called "Unexpected label ***(different numbers)". I just change the {numlabels} and the path of imgs/labels, could you please give me some instructions? Thank you~~

**Comment by codepujan** [comment]
@SJTUsuperxu  Hello , Just to follow up  this late . I ran into the same problem ( for training 2 classes ) , were you able to get through it , If so , Could you explain me your pipeline on how were you able to train it in 2 classes. It would be a great help !!
